molecule/source-upgrade/molecule.yml

```diff
@@ -39,6 +39,17 @@ provisioner:
     name: ansible-lint
 scenario:
   name: source-upgrade
+  test_sequence:
+    - lint
+    - dependency
+    - syntax
+    - create
+    - prepare
+    - converge
+    # - idempotence
```

Comment:
We need the comment from tox.ini here ("pip installing from source always replaces the installed version")